nservicebus/hosting/custom-configuration-providers.md

```diff
@@ -17,6 +17,7 @@ snippet: CustomConfigProvider
 
 Adding the code above is enough since NServiceBus will automatically use it found in the types scanned.
 
+NOTE: if multiple instances of `IProvideConfiguration<T>` for the same configuration sections `T` are found, only one instance will be used.
```

Comment:
@SimonCropp - good point. Can we document this for the time being and open an issue in the core to address this?